models/user.js

```diff
@@ -15,7 +15,7 @@ module.exports = (sequelize, DataTypes) => {
   {
     instanceMethods: {
```

Comment:
So cool that you created functions at the model level. I wonder if it's necessary to specify instanceMethods or you could directly define the function, making it less nested-like :)